client/src/frontend/components/composed/EntityHero/parts/Title/styles.js

```diff
@@ -0,0 +1,60 @@
+import styled from "@emotion/styled";
+import { respond } from "theme/styles/mixins";
+import { breakpoints } from "theme/styles/variables/media";
+import { transientOptions } from "helpers/emotionHelpers";
+
+const BREAKPOINT = breakpoints[60];
+
+export const Wrapper = styled("header", transientOptions)`
+  position: relative;
+  z-index: 50;
+  margin-bottom: 24px;
```

Comment:
This is actually supposed to separate the subtitle from the creators. The big gap we're seeing in Storybook looks like it's actually coming from `Meta` because there are no creators.